storm-server/src/test/java/org/apache/storm/scheduler/resource/strategies/scheduling/TestLargeCluster.java

```diff
@@ -62,12 +62,17 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.TreeMap;
+import java.util.concurrent.atomic.AtomicInteger;
 
 @ExtendWith({NormalizedResourcesExtension.class})
 public class TestLargeCluster {
     private static final Logger LOG = LoggerFactory.getLogger(TestLargeCluster.class);
 
-    public static final String TEST_CLUSTER_NAME = "largeCluster01";
+    public static final String TEST_CLUSTER_01 = "largeCluster01";
+    public static final String TEST_CLUSTER_02 = "largeCluster02";
+    public static final String TEST_CLUSTER_03 = "largeCluster03";
+
+    public static final String TEST_CLUSTER_NAME = TEST_CLUSTER_02;
```

Comment:
Earlier test was structured to test only one large cluster - since the cluster01 and cluster02 takes about two minutes each. Testing all will make this code easier to understand - especially with the supervisors being created in uniform manner as per your suggestion below.